### ISSUE TITLE
Drawer Key Tooltip is broken in Fullscreen and GUI Scale between 1-3

### ISSUE THREAD

**Issue by davpheol** [Nomi-CEu/Nomi-CEu#1050]
### Nomi CEu Version

1.7.1

### Launcher

Prism Launcher

### Configurations or Mods Changed

Fresh instance, nothing changed

### Environment

Singleplayer

### Mode

Normal Mode

### What Happened

The tooltip of the Drawer Key from Storage Drawers is broken when the game is in fullscreen and the gui scale is between 1-3.
![grafik](https://github.com/user-attachments/assets/bee66990-a389-4a2d-b7d2-3ffb24ad7886)

### Expected Behavior

It should probably look like this in Fullscreen
![grafik](https://github.com/user-attachments/assets/683518bd-5f11-4e9f-8ab6-150dc4ad4060)

### Reproduction Steps

1. Put the game in Fullscreen
2. Have the GUI Scale between 1-3 (Its fine on 4 and 5 )
3. Look at the tooltip for Drawer Key from Storage Drawers

**Comment by IntegerLimit** [comment]
Does this happen before 1.7.1?

**Comment by IntegerLimit** [comment]
Yes, it does. This is actually a rather widespread problem, (see Programmed Circuit Card) probably to do either with tooltip rendering not taking into account line endings in single tooltips, or issues with mac/linux vs windows line endings.

**Comment by davpheol** [comment]
Yeah i tested it on 1.0.1 of the pack and many other versions and the issue is still there. Probably a Windows Issue then.

**Comment by IntegerLimit** [comment]
No, I have the issue on Mac as well, so not sure. I was thinking it was because the line endings were designed for Windows.

Will see if I can do anything about this. There is a very simple way to fix it, but that would require replacing the tooltip, so I would rather a general fix.
